data/json/items/containers.json

```diff
@@ -112,7 +112,10 @@
         "to_hit":-1,
         "material":"steel",
         "contains":1,
-        "rigid": true
+        "rigid": true,
+        "watertight": true,
+        "seals": true,
+        "qualities": [["CONTAIN", 1]]
```

Comment:
Plastic bowl I think is supposed to be tupperware with a lid, I'm not sure about the clay bowl but it probably also shouldn't have a lid.